projects/js-packages/image-guide/src/MeasurableImage.ts

```diff
@@ -11,15 +11,19 @@ export class MeasurableImage {
 	readonly node: HTMLElement | HTMLImageElement;
 	private getURLCallback: SourceCallbackFn;
 
+	public fetch = fetch;
+
 	/**
 	 * Constructor.
 	 *
 	 * @param {HTMLElement | HTMLImageElement} node -  The DOM Element that contains the image.
 	 * @param {SourceCallbackFn} getURL             -  A function that takes in the node and returns the URL of the image.
+	 * @param {(input: string, init?: Array) => Promise<Response>} fetchFn                               -  A function that fetches a URL and returns a Promise.
```

Comment:
Why does the linter like you more than me? If I step out of line, the thing screams at me. :) 

The internal spacing to line up the descriptions on this comment is kinda weird. Minor nitpick, I know.